wordpress-mobile/WordPress-iOS
# Fix crash in Japanese locale

Fixes #12331.

This was solved together with @etoledom and @loremattei. 

I was able to reproduce this by navigating to a site with a name “To The Moon”. It seems to be the translation of the string:

```
"%@, at %@"
```

The current Japanese translation is:

```
"%1$s @ %2$s"
```

Changing it to this fixes it:

```
"%1$@ @ %2$@“
```

## Testing

1. Set the device language to Japanese.
2. Open the Site List.
3. Navigate to a site that can reproduce the crash described above.
4. Confirm it doesn't crash.

## Reviewing 

Only 1 reviewer is needed but anyone can review. 

## Release Notes
